openbb_platform/providers/coingecko/tests/test_coingecko_integration.py

```diff
@@ -0,0 +1,228 @@
+"""Integration tests for CoinGecko provider."""
+
```

Comment:
Integration tests go with the router extensions and should follow the established pattern of adding the parameters set to the interface tests where endpoints already exist. "./extensions/crypto/integration"